src/libs/actions/Report.js

```diff
@@ -907,6 +907,7 @@ function editReportComment(reportID, originalReportAction, textForNewComment) {
     const autolinkFilter = {filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')};
     const htmlForNewComment = parser.replace(textForNewComment, autolinkFilter);
     const originalMessageHTML = parser.replace(originalReportAction.message[0].html, autolinkFilter);
+    const parsedOriginalCommentHTML = parser.replace(parser.htmlToMarkdown(originalCommentHTML), autolinkFilter);
```

Comment:
```suggestion
    const originalMessageHTML = parser.replace(parser.htmlToMarkdown(originalReportAction.message[0].html), autolinkFilter);
```

Reply:
Keeping it as close to the code change here, except with the `parser.htmlToMarkdown` call